src/main/java/fmartin1/pokemon/PokemonType.java

```diff
@@ -0,0 +1,33 @@
+package fmartin1.pokemon;
+
+import fmartin1.common.NamedAPIResource;
+import fmartin1.pokemon.type.Type;
+
+public class PokemonType {
+    private int slot;
```

Comment:
What's the purpose of `slot` ?

Reply:
And for class variables use `_variableName` to avoid the usage of `this.variableName`